src/System.Private.Xml/tests/XmlNodeReader/System.Xml.XmlNodeReader.Tests/XmlNodeReaderMiscTests.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System.Collections.Generic;
+using Xunit;
+
+namespace System.Xml.Tests
+{
+    public class XmlNodeReaderMiscTests
+    {
+        [Fact]
+        public void NodeReaderCloseWithEmptyXml()
+        {
+            var nodeReader = new XmlNodeReader(new XmlDocument());
+            nodeReader.Close();
+            Assert.Equal(ReadState.Closed, nodeReader.ReadState);
+        }
+
+        [Fact]
+        public void NodeReaderSkipWithSimpleXml()
+        {
+            string xml = "<root atri='val'><child /></root>";
```

Comment:
This pattern 
```
+            string xml = "<root atri='val'><child /></root>";
+            var document = new XmlDocument();
+            document.LoadXml(xml);
+            var nodeReader = new XmlNodeReader(document);
```
might be worth extracting into a method that takes a string and returns a node reader.